src/email-templates/internal/campaign.created.hbs

```diff
@@ -1,3 +1,3 @@
 Campaign {{ campaign.name }} was created.
 
-Click here to view the campaign: {{ link-to account.uri "campaign" campaign.hash }}
+Click here to view the campaign: {{ link-to account.uri "app/campaign" campaign.id }}
```

Comment:
Where does this ultimately link to?

Reply:
E.g; https://allured.native-x.parameter1.com/app/campaign/123461235